tensorflow/core/platform/file_system.cc

```diff
@@ -259,4 +261,50 @@ Status FileSystem::RecursivelyCreateDir(const string& dirname) {
   return Status::OK();
 }
 
+Status FileSystem::CopyFile(const string& src, const string& target) {
+  uint64 size;
+  Status s = GetFileSize(src, &size);
```

Comment:
Use TF_RETURN_IF_ERROR(GetFileSize(src, &size)); instead. Here and elsewhere...

Reply:
Thanks @rohan100jain, done.